app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    people_list = [Person(person["name"], person["age"]) for person in people]
+
+    for i, person in enumerate(people):
+        if "wife" in person and person["wife"] is not None:
```

Comment:
you can use dictionary method `get`. It is safe method, 'cause in case if there is no such key in dictionary it will return `None`. That's exactly what u need, I guess

Reply:
emm.. what about `get()` method ?